insta/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './App.scss';
+
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/PostContainer';
+// import AddComment from './components/CommentSection/AddComment';
+import SearchBar from './components/SearchBar/SearchBar';
+
+class App extends React.Component {
+  state = {
+    dummy: dummyData
+  };
+  
+  render(){
+    return (
+      <div className="App">
+        <SearchBar /> 
+        <PostContainer dummy={this.state.dummy}/> 
```

Comment:
Great job passing down data to the lower component. Remember you can just as easily map over the data here.  Its not wrong. Just giving you choices to use when you code.